.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Codeowners file for auto-assignment of experienced developers for code review
+
+*	@kashomon @mhoak
+
+# Auto-assignment of relevant developers for code review
+
+core/sgf/	@kashomon @mhoak @TrevorPeters
```

Comment:
Are directory assignment rules inherited? 

I.e., does this also get the * rules by default? In which case we could just put TrevorPeters?

Reply:
My understanding is that the lowest level takes precedence, in which case only TrevorPeters would be assigned.